phac-nml/irida-next
# Attachment destroy service

## What does this PR do and why?
This PR creates the destroy service for attachments, moving the destroy functionality away from the attachments controller. In addition to the destroy service, deletion of paired sample attachments is now also fixed whereby the shared delete button on the samples attachment page for paired attachments will now delete both associated attachments. 

## Screenshots or screen recordings
[Screencast from 2023-11-03 09:23:46 AM.webm](https://github.com/phac-nml/irida-next/assets/82407232/49204e67-e806-486f-921c-007aa0773132)

## How to set up and validate locally
Upload both single and paired attachments to a sample and delete, ensuring delete functionality is handled as expected.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
